### PR TITLE
[Csp] Covering the model classes by Unit Tests

### PR DESCRIPTION
**Pull request by eduard13**
### Description (*)
This PR covers the following model classes by Unit Tests:
- Magento\Csp\Model\PolicyRendererPool
- Magento\Csp\Model\Mode\ConfigManager

### Fixed Issues (if relevant)
N/A

### Manual testing scenarios (*)
N/A

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/Magento/Csp/Test/Unit/Model/Mode/ConfigManagerTest.php [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use Magento\Framework\App\State;
+use Magento\Framework\TestFramework\Unit\Helper\ObjectManager;
+use Magento\Store\Model\Store;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use RuntimeException;
+
+/**
+ * Class ConfigManagerTest
```
Comment (lbajsarowicz) [inline app/code/Magento/Csp/Test/Unit/Model/Mode/ConfigManagerTest.php]: This annotation is not necessary.

**Review by dmytro-ch — APPROVED** [review]

File: app/code/Magento/Csp/Test/Unit/Model/Mode/ConfigManagerTest.php [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+        );
+    }
+
+    /**
+     * Test throwing an exception for non storefront or admin areas
+     *
+     * @return void
+     *
+     * @expectedExceptionMessage CSP can only be configured for storefront or admin area
+     * @expectedException RuntimeException
```
Comment (lbajsarowicz) [inline app/code/Magento/Csp/Test/Unit/Model/Mode/ConfigManagerTest.php]: Use `$this->expectException` and `$this->expectExceptionMessage` instead.
Annotations are going to be deprecated in further versions of PHPUnit and finally removed in PHPUnit 9.

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch, thank you for the review.
[ENGCOM-6725](https://jira.corp.magento.com/browse/ENGCOM-6725) has been created to process this Pull Request
:eight_spoked_asterisk: @dmytro-ch, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

File: app/code/Magento/Csp/Test/Unit/Model/PolicyRendererPoolTest.php [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+use RuntimeException;
+
+/**
+ * Class PolicyRendererPoolTest
+ *
+ * Test for \Magento\Csp\Model\PolicyRendererPool
+ */
+class PolicyRendererPoolTest extends TestCase
+{
+    private const STUB_POLICY_ID = 'header';
```
Comment (lbajsarowicz) [inline app/code/Magento/Csp/Test/Unit/Model/PolicyRendererPoolTest.php:24]: :+1:

**Review by lbajsarowicz — CHANGES_REQUESTED** [review]
Annotations are discouraged in PHPUnit and finally in version 9 are removed.

File: app/code/Magento/Csp/Test/Unit/Model/Mode/ConfigManagerTest.php (hunk 2 above)
Comment (eduard13) [reply app/code/Magento/Csp/Test/Unit/Model/Mode/ConfigManagerTest.php]: Hi @lbajsarowicz, that's makes sense.
Thanks.

**Review by lbajsarowicz — APPROVED** [review]
Great job!

**Comment by magento-engcom-team** [comment]
Hi @lbajsarowicz, thank you for the review.
[ENGCOM-6725](https://jira.corp.magento.com/browse/ENGCOM-6725) has been created to process this Pull Request
:eight_spoked_asterisk: @lbajsarowicz, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by engcom-Alfa** [comment]
:heavy_check_mark: QA Passed

**Pull request merged**